src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSidebar/MyCollectionArtworkSidebarMetadata.tsx

```diff
@@ -0,0 +1,117 @@
+import { Box, Spacer, Text } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { MyCollectionArtworkSidebarMetadata_artwork } from "__generated__/MyCollectionArtworkSidebarMetadata_artwork.graphql"
+
+export interface MyCollectionArtworkSidebarMetadataProps {
+  artwork: MyCollectionArtworkSidebarMetadata_artwork
+}
+
+export const MyCollectionArtworkSidebarMetadata: React.FC<MyCollectionArtworkSidebarMetadataProps> = ({
+  artwork,
+}) => {
+  const {
+    artistNames,
+    artworkLocation,
+    attributionClass,
+    category,
+    date,
+    dimensions,
+    medium,
+    pricePaid,
+    provenance,
+    title,
+  } = artwork
+
+  return (
+    <>
+      <Text as="h1" variant="lg-display">
+        {artistNames}
+      </Text>
+      <Text as="h1" variant="lg-display" color="black60" mb={[0.5, 0]}>
+        <i>{title?.trim()}</i>
+        {date && date.replace(/\s+/g, "").length > 0 && ", " + date}
+      </Text>
+
+      <Spacer m={[1, 2]} />
+
+      <MetadataField label="Medium" value={category} />
+      <MetadataField label="Materials" value={medium} />
+      <MetadataField
+        label="Rarity"
+        value={attributionClass?.shortDescription}
+      />
+
+      <Box mb={[1, 0.5]} display="flex" flexDirection={["column", "row"]}>
+        <Text variant="sm" minWidth={190} mr={2}>
+          Size
+        </Text>
+
+        {dimensions ? (
+          <Box>
+            {!!dimensions.in && (
+              <Text variant="sm" color="black60">
+                {dimensions.in}
+              </Text>
+            )}
+            {!!dimensions.cm && (
+              <Text variant="sm" color="black60">
+                {dimensions.cm}
+              </Text>
+            )}
+          </Box>
+        ) : (
+          <Text variant="sm" color="black60">
+            ----
+          </Text>
+        )}
+      </Box>
+
```

Comment:
You could use [`white-space: pre-line`](https://developer.mozilla.org/en-US/docs/Web/CSS/white-space) here to use your existing `MetadataField` component.

```suggestion
<MetadataField
        label="Size"
        value={[artwork.dimensions?.in, artwork.dimensions?.cm]
          .filter(d => d)
          .join("\n")}
      />
```

Reply:
You'd just need to wrap `value` in a styled `Text` component like this:

```
const WrappedText = styled(Text)`
  white-space: pre-line;
`
```